odbc/src/api/definitions.rs

```diff
@@ -110,3 +110,256 @@ pub enum UseBookmarks {
+    TimedateFunctions = 52,
+    // SQL_CONVERT_BIGINT
+    ConvertBigInt = 53,
+    // SQL_CONVERT_BINARY
+    ConvertBinary = 54,
+    // SQL_CONVERT_BIT
+    ConvertBit = 55,
+    // SQL_CONVERT_CHAR
+    ConvertChar = 56,
+    // SQL_CONVERT_DATE
+    ConvertDate = 57,
+    // SQL_CONVERT_DECIMAL
+    ConvertDecimal = 58,
+    // SQL_CONVERT_DOUBLE
+    ConvertDouble = 59,
+    // SQL_CONVERT_FLOAT
+    ConvertFloat = 60,
+    // SQL_CONVERT_INTEGER
+    ConvertInteger = 61,
+    // SQL_CONVERT_LONGVARCHAR
+    ConvertLongVarChar = 62,
+    // SQL_CONVERT_NUMERIC
+    ConvertNumeric = 63,
+    // SQL_CONVERT_REAL
+    ConvertReal = 64,
+    // SQL_CONVERT_SMALLINT
+    ConvertSmallInt = 65,
+    // SQL_CONVERT_TIME
+    ConvertTime = 66,
+    // SQL_CONVERT_TIMESTAMP
+    ConvertTimestamp = 67,
+    // SQL_CONVERT_TINYINT
+    ConvertTinyInt = 68,
+    // SQL_CONVERT_VARBINARY
+    ConvertVarBinary = 69,
+    // SQL_CONVERT_VARCHAR
+    ConvertVarChar = 70,
+    // SQL_CONVERT_LONGVARBINARY
+    ConvertLongVarBinary = 71,
+    // SQL_GETDATA_EXTENSIONS
+    GetDataExtensions = 81,
+    // SQL_COLUMN_ALIAS
+    ColumnAlias = 87,
+    // SQL_GROUP_BY
+    GroupBy = 88,
+    // SQL_ORDER_BY_COLUMNS_IN_SELECT
+    OrderByColumnsInSelect = 90,
+    // SQL_OWNER_USAGE
+    OwnerUsage = 91,
+    // SQL_CATALOG_USAGE
+    CatalogUsage = 92,
+    // SQL_SPECIAL_CHARACTERS
+    SpecialCharacters = 94,
+    // SQL_MAX_COLUMNS_IN_GROUP_BY
+    MaxColumnsInGroupBy = 97,
+    // SQL_MAX_COLUMNS_IN_ORDER_BY
+    MaxColumnsInOrderBy = 99,
+    // SQL_MAX_COLUMNS_IN_SELECT
+    MaxColumnsInSelect = 100,
+    // SQL_TIMEDATE_ADD_INTERVALS
+    TimedateAddIntervals = 109,
+    // SQL_TIMEDATE_DIFF_INTERVALS
+    TimedateDiffIntervals = 110,
+    // SQL_CATALOG_LOCATION
+    CatalogLocation = 114,
+    // SQL_SQL_CONFORMANCE
+    SqlConformance = 118,
+    // SQL_CONVERT_WCHAR
+    ConvertWChar = 122,
+    // SQL_CONVERT_WLONGVARCHAR
+    ConvertWLongVarChar = 125,
+    // SQL_CONVERT_WVARCHAR
+    ConvertWVarChar = 126,
+    // SQL_ODBC_INTERFACE_CONFORMANCE
+    OdbcInterfaceConformance = 152,
+    // SQL_SQL92_PREDICATES
+    Sql92Predicates = 160,
+    // SQL_SQL92_RELATIONAL_JOIN_OPERATORS
+    Sql92RelationalJoinOperators = 161,
+    // SQL_AGGREGATE_FUNCTIONS
+    AggregateFunctions = 169,
+    // SQL_CONVERT_GUID
+    ConvertGuid = 173,
+    // SQL_RETURN_ESCAPE_CLAUSE
+    ReturnEscapeClause = 180,
+    // SQL_API_SQLFETCHSCROLL
+    ApiSqlFetchScroll = 1021,
+    // SQL_CATALOG_NAME
+    CatalogName = 10003,
+    // SQL_MAX_IDENTIFIER_LEN
+    MaxIdentifierLen = 10005,
+}
+
+pub const SQL_CB_NULL: u16 = 0x0000;
+pub const SQL_U16_ZERO: u16 = SQL_CB_NULL;
+pub const SQL_CL_START: u16 = 0x0001;
+pub const SQL_U32_ZERO: u32 = 0;
+pub const SQL_OIC_CORE: u32 = 0x00000001;
+pub const SQL_SC_SQL92_ENTRY: u32 = 0x00000001;
+pub const SQL_INFO_Y: &str = "Y";
+
+// SQL_CONVERT_FUNCTIONS bitmask
+pub const SQL_FN_CVT_CAST: u32 = 0x00000002;
```

Comment:
In general here and below I only include the values that we actually use. The alternative would be including _all_ of the values for a certain set (for example, here, the `SQL_FN_CVT_CONVERT: u32 = 0x00000001;` value) and marking it with a rust attribute to ignore dead/unused code. Not sure that's worth it though! I think this current implementation is cleaner since it avoids dead/unused code.